src/style.css

```diff
@@ -1,3 +1,96 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --header-height: 60px;
+  --font-size: 12px;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: auto;
+  max-width: 1200px;
+  min-width: 1024px;
+  width: 100%;
```

Comment:
It's not recommended to fix container sizes with `min-width` and `max-width` unless it's a specific requirement. Let the content dictate the size of the container to avoid overflow or unnecessary scroll bars.